nixos/modules/services/misc/nesthub.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nesthub;
+in {
+  options = {
+    services.nesthub = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the Nesthub service.";
+      };
+      configPath = mkOption {
+        type = types.str;
+        default = "/var/lib/nesthub/config.json";
+        description = "Path to the config file.";
+      };
```

Comment:
I don't know what nesthub is, nor how it works - but is the user expected to write their own config file to that path, or does nesthub generate one and place it there? (you have to configure it through the application)